Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -595,14 +582,14 @@ void ITSClusterTask::addObject(TObject* aObject)
   } else
     mPublishedObjects.push_back(aObject);
 }
-
+/*
```

Comment:
let's remove this function if you don't need it anymore

Reply:
Done